lib/concatSeries.js

```diff
@@ -19,4 +19,4 @@ import doSeries from './internal/doSeries';
  * containing the concatenated results of the `iteratee` function. Invoked with
  * (err, results).
  */
-export default doSeries(concat);
```

Comment:
Yup. From the history, it looks like all of the other `doSeries` were converted to `doLimit(<function_name>Limit, 1);` when the `*Series` functions were implemented in terms of `*Limit` [here](https://github.com/caolan/async/commit/acd656ddfda189fadfe4505f2b9e3eae6512f4dc).